### PR TITLE
Honda : 2023 Pilot Car Port

### PR DESCRIPTION
**Pull request by vanillagorillaa**
**Checklist**
- [ ] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [X] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot: f9c43864cf057d05|2024-01-15--23-01-20
- [X] route with stock system: f9c43864cf057d05|2024-01-13--14-08-31
- [X] Harness : Bosch C

First port for a new CAN-FD Honda

DBC Notes: 
Using accord DBC, these messages are missing. Will have to check if they are on different ID's now
229 BOSCH_SUPPLEMENTAL_1
232 BRAKE_HOLD
316 GAS_PEDAL
490 VEHICLE_DYNAMICS
545 XXX_16
597 ROUGH_WHEEL_SPEED
892 CRUISE_PARAMS
927 RADAR_HUD
1029 DOOR_STATUS
13274 LKAS_HUD_A
13275 LKAS_HUD_B

### CODE REVIEW

File: selfdrive/car/torque_data/params.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ HONDA HRV 2019: [2.0661212805710205, 0.7521343418694775, 0.17760375789242094]
 HONDA INSIGHT 2019: [1.5201671214069354, 0.5660229120683284, 0.25808042580281876]
 HONDA ODYSSEY 2018: [1.8774809275211801, 0.8394431662987996, 0.2096978613792822]
 HONDA PILOT 2017: [1.7262026201812795, 0.9470005614967523, 0.21351430733218763]
+HONDA PILOT 2023: [1.7262026201812795, 0.9470005614967523, 0.21351430733218763]
```
Comment (sshane) [inline selfdrive/car/torque_data/params.yaml:25]: You can use the substitute file if you're just copying the values

Comment (vanillagorillaa) [reply selfdrive/car/torque_data/params.yaml]: Thanks! Added!

File: selfdrive/car/honda/interface.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     CAN = CanBus(ret, fingerprint)
 
     if candidate in HONDA_BOSCH:
-      cfgs = [get_safety_config(car.CarParams.SafetyModel.ford)]
```
Comment (vanillagorillaa) [inline selfdrive/car/honda/interface.py]: @sshane You had initially set this to ford, assuming this was just a copy paste from ford/interface with the same cod?

**Comment by vanillagorillaa** [comment]
0x33D, LKAS_HUD, on this platform is 8 bytes, not 5 like previous Bosch vehicles

**Comment by sshane** [comment]
First Honda that doesn't require the OBD port! 🎉

**Comment by sshane** [comment]
We've moved the car interfacing code to our [`opendbc`](https://github.com/commaai/opendbc) repository, which is now the new home for car ports and fingerprints. Please re-open your pull request against [`opendbc`](https://github.com/commaai/opendbc) at your convenience by using this command below. This will transform all changes under `selfdrive/car/` to `opendbc_repo/opendbc/car/`. Make sure you have initialized submodules beforehand and have checked out the latest `opendbc` commit.

```bash
PR_NUMBER=33045
curl -L https://github.com/commaai/openpilot/pull/$PR_NUMBER.patch | sed -e 's/selfdrive\/car/opendbc_repo\/opendbc\/car/g' | git apply -v --reject
```

Simply replace the PR number with your own. Once done, add the files, fix any conflicts, and open a new PR. Alternatively, you may start a new PR from scratch if that is easier for you.

**Pull request closed without merging**